int64 instance
int64 proposal_num
string proposer_id
int64 proposed_value